src/Features/LanguageServer/Protocol/LspOptionsStorage.cs

```diff
@@ -41,5 +42,10 @@ internal sealed class LspOptionsStorage
         /// Flag indicating whether or not test and debug code lens items should be returned.
         /// </summary>
         public static readonly PerLanguageOption2<bool> LspEnableTestsCodeLens = new("dotnet_enable_tests_code_lens", defaultValue: true, group: s_codeLensOptionGroup);
+
+        /// <summary>
+        /// <see langword="true"/> to sort imports as part of LSP's Format Document handler; otherwise, <see langword="false"/>.
+        /// </summary>
+        public static readonly PerLanguageOption2<bool> LspFormattingSortImports = new("dotnet_formatting_sortImports", defaultValue: false, group: s_formattingOptionGroup);
```

Comment:
the mix of camelCast and snake_case is strange.  why not `sort_imports`?

Reply:
⏱️ Will update

Reply:
➡️ Updated